InvenTree/InvenTree/settings.py

```diff
@@ -912,3 +913,21 @@ def _is_true(x):
 PLUGIN_TESTING_SETUP = get_setting('PLUGIN_TESTING_SETUP', False)  # load plugins from setup hooks in testing?
 PLUGIN_RETRY = get_setting('PLUGIN_RETRY', 5)  # how often should plugin loading be tried?
 PLUGIN_FILE_CHECKED = False                    # Was the plugin file checked?
+
+# user interface customization values
+CUSTOMIZE = get_setting(
+    'INVENTREE_CUSTOMIZE',
+    CONFIG.get('customize', {}),
+    {}
+)
+
+CUSTOM_LOGO = get_setting(
```

Comment:
This option will (by default) throw a warning, right? as 'INVENTREE_LOGO' will not exist in the media dir.

Should the default value be an empty string (''), or the line commented out?

Reply:
Not exactly. As custom logo is false by default it is not checked. So everything keeps functioning as it used to be.

Reply:
@SchrodingersGat can I mark this as resolved?